src/record.c

```diff
@@ -348,6 +349,8 @@ int janus_recorder_save_frame(janus_recorder *recorder, char *buffer, uint lengt
 		json_object_set_new(info, "c", json_string(recorder->codec));					/* Media codec */
 		if(recorder->fmtp)
 			json_object_set_new(info, "f", json_string(recorder->fmtp));				/* Codec-specific info */
+		if(recorder->description)
+			json_object_set_new(info, "desc", json_string(recorder->description));		/* Stream description */
```

Comment:
All other properties have a one-char attribute name, so I'd follow the same "rule": maybe `d` if it's not used yet?

Reply:
Fixed, there is another value under the name "or", which is why I didn't realize there was a naming convention here.